workflows/rnaseq.nf

```diff
@@ -768,6 +772,15 @@ workflow RNASEQ {
             ch_versions = ch_versions.mix(DUPRADAR.out.versions.first())
         }
 
+        if (params.ngscheckmate_bed) {
+            BAM_NGSCHECKMATE (
```

Comment:
Is there any way the output could be baked into the MultiQC report?

Reply:
That has been suggested over in sarek too, something I could consider.